manifests/cluster-logging.package.yaml

```diff
@@ -1,5 +1,5 @@
 #! package-manifest: ./deploy/chart/catalog_resources/rh-operators/clusterlogging.v4.1.0.clusterserviceversion.yaml
 packageName: cluster-logging
 channels:
-- name: 4.2
+- name: "4.2"
```

Comment:
We definitely need this fixed asap

Reply:
What action are you asking me to take here?